src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -148,6 +148,109 @@ void listSitesWithKeys(Vertx vertx, VertxTestContext testContext) {
         });
     }
 
+    @Test
+    void getSiteWithStringKeyKeys(Vertx vertx, VertxTestContext testContext){
```

Comment:
What do you mean by `Keys` here? seems like `Id` might be more suitable. e.g. `getSiteWithStringId`

Reply:
Yeah I did... Sorry I'll fix all of those.